src/NuGet/Microsoft.Orleans.Client.nuspec

```diff
@@ -18,9 +18,9 @@
     <copyright>Copyright Microsoft 2015</copyright>
     <tags>Orleans Cloud-Computing Actor-Model Actors Distributed-Systems C# .NET</tags>
     <dependencies>
-      <dependency id="Microsoft.Orleans.Core" version="1.0.3" />
-      <dependency id="Microsoft.Orleans.OrleansAzureUtils" version="1.0.3" />
-      <dependency id="Microsoft.Orleans.OrleansProviders" version="1.0.3" />
+      <dependency id="Microsoft.Orleans.Core" version="$version$" />
```

Comment:
Well, i think we can *guess* about backward compatibility, but can't be 100% sure unless it is explicitly tested. [Not something anyone is planning to do any time soon, AFAIK]

So this would mean >= 1.0.5, which would be the new versions.
``` xml
<dependency id="Microsoft.Orleans.Core" version="1.05" />
```

Are you actually saying that we should leave this at ```1.0.3``` instead?